tools/onebranch/onebranch.vcxproj.filters

```diff
@@ -17,4 +17,9 @@
       <Extensions>rc;ico;cur;bmp;dlg;rc2;rct;bin;rgs;gif;jpg;jpeg;jpe;resx;tiff;tif;png;wav;mfcribbon-ms</Extensions>
     </Filter>
   </ItemGroup>
+  <ItemGroup>
+    <ResourceCompile Include="$(SolutionDir)resource\ebpf_resource.rc">
```

Comment:
Why does this file need ebpf_resource.rc added whereas the others needed it deleted?
I.e., why can't the same solution work for both sets of files?